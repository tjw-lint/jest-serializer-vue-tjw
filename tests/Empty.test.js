import { mount, shallowMount } from '@vue/test-utils';
import Empty from './components/Empty.vue';

describe('Empty.vue', () => {
  test('Properly serializes a shallowly-rendered wrapper', () => {
    const wrapper = shallowMount(Empty);

    expect(wrapper)
      .toMatchSnapshot();
  });

  test('Properly serializes a fully-mounted wrapper', () => {
    const wrapper = mount(Empty);

    expect(wrapper)
      .toMatchSnapshot();
  });
});
