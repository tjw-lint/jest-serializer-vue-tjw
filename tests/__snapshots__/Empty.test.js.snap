// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty.vue Properly serializes a fully-mounted wrapper 1`] = ``;

exports[`Empty.vue Properly serializes a shallowly-rendered wrapper 1`] = ``;
